src/components/progress-tracker/progress-tracker.spec.tsx

```diff
@@ -18,17 +18,19 @@ import useResizeObserver from "../../hooks/__internal__/useResizeObserver";
 
 jest.mock("../../hooks/__internal__/useResizeObserver");
 
+const mockedUseResizeObserver = useResizeObserver as jest.Mock<() => void>;
+
 describe("ProgressTracker", () => {
-  let wrapper;
+  let wrapper: ReactWrapper;
 
   const originalOffsetHeight = Object.getOwnPropertyDescriptor(
     HTMLElement.prototype,
     "offsetHeight"
-  );
+  ) as PropertyDescriptor | ThisType<unknown>;
```

Comment:
I confess I don't really understand what the `ThisType` is doing here - but I tried removing it, both here and on line 33, and VSCode isn't complaining at me. So it looks like it's not needed.

Reply:
The type expected was `PropertyDescriptor | ThisType<any>`.  This is a leftover though, now it's just being cast as `PropertyDescriptor` to resolve the type issue `ThisType` isn't needed. Good spot.